Source/Houdini/Houdini.cs

```diff
@@ -1439,7 +1439,7 @@ protected void DebugRefutedCandidates(Implementation curFunc, List<Counterexampl
         }
 
         DateTime end = DateTime.UtcNow;
-        xmlRefuted.WriteEndMethod("errors", end, end.Subtract(start));
+        xmlRefuted.WriteEndMethod("errors", end, end.Subtract(start), 0);
```

Comment:
Note I'm NOT confident in this since I don't yet understand how Houdini works.

Reply:
The `resourceCount` argument is now nullable, and no `resourceCount` attribute gets added to the XML if it's null. This should lead to no changes in Houdini XML output.